Library/Homebrew/cmd/audit.rb

```diff
@@ -1234,4 +1235,5 @@ def audit_urls
   def problem(text)
     @problems << text
   end
+
```

Comment:
Not at all. This is stuff I care about too; it's just slipping past me in my edit / amend / comment / push cycle. I need to figure out a better work flow for this.